scripts/compile.mjs

```diff
@@ -56,3 +56,12 @@ await fs.promises.writeFile(
     <script src="./hglib.min.js"></script>
   `)
 );
+
```

Comment:
What is this file?

Reply:
This file is executed within `npm run compile`. First we build the library with Vite, and then this script adapts those outputs to create identical outputs to the previous from webpack. There is probably a way to do this all with a complicated Vite config, but I think this is much easier to read/understand.